livre/core/data/VolumeDataSource.cpp

```diff
@@ -114,7 +111,7 @@ void VolumeDataSource::unloadPlugins()
         detail::VolumeDataSource::PluginFactory::getInstance().unload( plugin );
 }
 
-ConstLODNodePtr VolumeDataSource::getNode( const NodeId nodeId ) const
+LODNode VolumeDataSource::getNode( const NodeId nodeId ) const
```

Comment:
NodeId is passed inconsistently though: sometimes by value and sometimes by const ref. Can be unified?